tests/phpunit/elementor/includes/test-utils.php

```diff
@@ -180,4 +180,14 @@ public function test_is_empty() {
 		$this->assertEquals( false, Utils::is_empty( [ 'key' => '0' ], 'key' ),
 			"[ 'key' => '0' ] is empty" );
 	}
+
+	public function test_remove_empty_array_recursive() {
```

Comment:
Can you test a situation that there is an array with a value inside and make sure that the value saved and not removed

Reply:
Handled.